### ISSUE TITLE
is listResourceRecords broken for truncated results?

### ISSUE THREAD

**Issue by leorossi** [chilts/nice-route53#9]
On my work project I have a zone with > 100 records and the first all of `listResourceRecords` returns me this

```
{ ResourceRecordSets: 
   [ REDACTED ],
  IsTruncated: true,
  NextRecordName: 'a.record.name.',
  NextRecordType: 'CNAME',
  MaxItems: '100' }
```
There is no `NextRecordIdentifier` so on the second call of `listRecourceRecords` `nextIdentifier = undefined` and I end up with such an error

```
There were 2 validation errors:
* UnexpectedParameter: Unexpected key 'Name' found in params
* UnexpectedParameter: Unexpected key 'Type' found in params
```

**Comment by leorossi** [comment]
I am working on a PR right now

**Comment by leorossi** [comment]
Merged #10
